net/dnscrypt-proxy/files/dnscrypt-proxy.init

```diff
@@ -33,7 +33,7 @@ create_config_file() {
     config_get      client_key      $1 'client_key'     ''
     config_get      syslog_prefix   $1 'syslog_prefix'  'dnscrypt-proxy'
     config_get      query_log_file  $1 'query_log_file' ''
-    config_get      log_level       $1 'log_level'	''
+    config_get      log_level       $1 'log_level'	'5'
```

Comment:
Also, the dnscrypt init.d file must have a value for log_level, it can't be left blank as in my first commit, or it crashes silently.  Hence why I added the default of 5.